pkg/validators/am0012_test.go

```diff
@@ -0,0 +1,47 @@
+package validators_test
+
+import (
+	"github.com/mt-sre/addon-metadata-operator/internal/testutils"
+	"github.com/mt-sre/addon-metadata-operator/pkg/types"
+	"github.com/mt-sre/addon-metadata-operator/pkg/utils"
+	"github.com/mt-sre/addon-metadata-operator/pkg/validators"
+)
+
+func init() {
+	TestRegistry.Add(TestAM0012{})
+}
+
+type TestAM0012 struct{}
+
+func (val TestAM0012) Name() string {
+	return validators.AM0012.Name
+}
+
+func (val TestAM0012) Run(mb types.MetaBundle) types.ValidatorResult {
+	return validators.AM0012.Runner(mb)
+}
+
+func (val TestAM0012) SucceedingCandidates() ([]types.MetaBundle, error) {
+	res, err := testutils.DefaultSucceedingCandidates()
```

Comment:
Moved the more specific validation testing to  https://github.com/mt-sre/addon-metadata-operator/blob/a435ed8c4f8a1fc08002586ef2b01a3fea38da90/pkg/utils/csvutils/csv_rbac_utils_test.go